guides/source/active_record_validations.md

```diff
@@ -521,7 +523,7 @@ constraints to acceptable values:
 
 NOTE: By default, `numericality` doesn't allow `nil` values. You can use `allow_nil: true` option to permit it.
 
-The default error message is _"is not a number"_.
+The default error message **when no options are specified** is _"is not a number"_.
```

Comment:
Sorry, I used **bold** to highlight my suggested change - I don't think it should actually appear this way in the guide.

```suggestion
The default error message when no options are specified is _"is not a number"_.
```

Reply:
Lol, it's cool. I force pushed an amendment.